elodie/filesystem.py

```diff
@@ -333,6 +361,10 @@ def process_file(self, _file, destination, media, **kwargs):
             print('%s is not a valid media file. Skipping...' % _file)
             return
 
+        original_checksum = self.process_checksum(_file, allow_duplicate)
+        if(original_checksum is None):
```

Comment:
I would add a log message here `log.info('Original checksum returned None for %s. Skipping...', _file)`.